configure.ac

```diff
@@ -1726,4 +1726,12 @@ Check '[$]0 --help' for available options
 
 fi
 ])
+
+AC_CONFIG_COMMANDS_POST([
+  # Disable PACKAGE_* symbols in main/php_config.h.in
+  $SED -e 's/^#undef PACKAGE_[^ ]*/\/\* & \*\//g' < $srcdir/main/php_config.h.in \
+    > $srcdir/main/php_config.h.in.tmp && \
+    mv $srcdir/main/php_config.h.in.tmp $srcdir/main/php_config.h.in
```

Comment:
Use `sed -i` instead?

Reply:
`-i` is not posix compliant option for sed :) Yes, I'd love to use the inplace directly also but it would need to be adjusted based on the system (BSD based and GNU based ones). Let me check...

Reply:
Ugh :( I guess it's fine then...